Avoid error output when libtool is Apple libtool.
Print errors to stderr not stdout.
This patch should be sent to the developers.
--- autogen.sh.orig	2022-11-15 09:40:57.000000000 -0600
+++ autogen.sh	2023-05-02 20:45:50.000000000 -0500
@@ -14,12 +14,12 @@
 #
 # See the GNU Lesser General Public License in COPYING.LGPL for more details.
 #
-if ! libtool --version > /dev/null
+if ! libtool --version >/dev/null 2>&1
 then
 	# Perhaps we are on a Mac
-	if ! glibtool --version > /dev/null
+	if ! glibtool --version >/dev/null 2>&1
 	then
-		echo "GNU Libtool is missing, please install it and fix the PATH to it."
+		echo "GNU Libtool is missing, please install it and fix the PATH to it." >&2
 		exit 1
 	else
 		export LIBTOOL=glibtool
